site/docs/2.3.1/api/scala/org/apache/spark/Accumulable.html

```diff
@@ -0,0 +1,706 @@
+              anchor_opt = '@' + anchor.substring(1);
+            window.location.href = url + '#' + hash + anchor_opt;
+         }
+   	  </script>
+    
+        </head>
+        <body class="type">
+      <div id="definition">
+        <img alt="Class" src="../../../lib/class_big.png" />
+        <p id="owner"><a href="../../package.html" class="extype" name="org">org</a>.<a href="../package.html" class="extype" name="org.apache">apache</a>.<a href="package.html" class="extype" name="org.apache.spark">spark</a></p>
+        <h1>Accumulable</h1><h3><span class="morelinks"><div>Related Doc:
+            <a href="package.html" class="extype" name="org.apache.spark">package spark</a>
+          </div></span></h3><span class="permalink">
+      <a href="../../../index.html#org.apache.spark.Accumulable" title="Permalink" target="_top">
+        <img src="../../../lib/permalink.png" alt="Permalink" />
+      </a>
+    </span>
+      </div>
+
+      <h4 id="signature" class="signature">
+      <span class="modifier_kind">
+        <span class="modifier"></span>
+        <span class="kind">class</span>
+      </span>
+      <span class="symbol">
+        <span class="name deprecated" title="Deprecated: (Since version 2.0.0) use AccumulatorV2">Accumulable</span><span class="tparams">[<span name="R">R</span>, <span name="T">T</span>]</span><span class="result"> extends <span class="extype" name="java.io.Serializable">Serializable</span></span>
+      </span>
+      </h4>
+      
+          <div id="comment" class="fullcommenttop"><div class="comment cmt"><p>A data type that can be accumulated, i.e. has a commutative and associative &quot;add&quot; operation,
+but where the result type, <code>R</code>, may be different from the element type being added, <code>T</code>.</p><p>You must define how to add data, and how to merge two of these together.  For some data types,
+such as a counter, these might be the same operation. In that case, you can use the simpler
+<a href="Accumulator.html" class="extype" name="org.apache.spark.Accumulator">org.apache.spark.Accumulator</a>. They won't always be the same, though -- e.g., imagine you are
+accumulating a set. You will add items to the set, and you will union two sets together.</p><p>Operations are not thread-safe.
+</p></div><dl class="paramcmts block"><dt class="tparam">R</dt><dd class="cmt"><p>the full accumulated data (result type)</p></dd><dt class="tparam">T</dt><dd class="cmt"><p>partial data that can be added in</p></dd></dl><dl class="attributes block"> <dt>Annotations</dt><dd>
+                <span class="name">@deprecated</span>
+              
+        </dd><dt>Deprecated</dt><dd class="cmt"><p><i>(Since version 2.0.0)</i> use AccumulatorV2</p></dd><dt>Source</dt><dd><a href="https://github.com/apache/spark/tree/v2.3.1/tmp/2.3.1/docs/core/src/main/scala/org/apache/spark/Accumulable.scala" target="_blank">Accumulable.scala</a></dd></dl><div class="toggleContainer block">
```

Comment:
https://github.com/apache/spark/tree/v2.3.1/tmp/2.3.1/docs/core/src/main/scala/org/apache/spark/Accumulable.scal should be https://github.com/apache/spark/blob/v2.3.1/core/src/main/scala/org/apache/spark/Accumulable.scala?

Reply:
Hmm, weird, all source links are wrong. Let me see if I figure out what happened, since this is all auto-generated.

Reply:
Seems to be SPARK-23732. Let me try the workaround there.